src/panels/OriginPanel/OriginPanelMenuButton.tsx

```diff
@@ -41,12 +42,14 @@ export function OriginPanelMenuButton({ eventHandlers }: Props) {
       aimName={aimNode?.name}
       isOpenSpaceReady={isReady}
       eventHandlers={eventHandlers}
+      isOpen={isOpen}
```

Comment:
I guess if we went by that change this would need to be extracted from redux an extra time